src/main/java/org/apache/commons/imaging/formats/tiff/TiffReader.java

```diff
@@ -412,7 +423,7 @@ public void read(final ByteSource byteSource, final Map<String, Object> params,
         readDirectories(byteSource, formatCompliance, listener);
     }
 
-    private TiffImageData getTiffRawImageData(final ByteSource byteSource,
+    public TiffImageData getTiffRawImageData(final ByteSource byteSource,
```

Comment:
Making methods that were private now public always need a short explanation, as these methods will be now part of the maintained API accessible by users, and we will have to have more care to change it.

Could you explain why these two methods in TiffReader are public and not private, please?

Reply:
I reverted these changes